addon/components/frost-popover.js

```diff
@@ -60,11 +62,16 @@ export default Component.extend(PropTypeMixin, {
     const handlerOut = this.get('handlerOut')
     if (handlerIn && handlerOut) {
       this._eventHandlerIn = (event) => {
+        run(() => {
```

Comment:
This doesn't seem right to me. The runloop invocation here is redundant. You should just run stopPropagation() without it.

Reply:
Ok, second thoughts that might work actually. I get a bit confused with the commit message. In the message you mentioned run does not work. Are you referring to another run?

Reply:
I have the same question/concern about why we don't want the event propagation to occur immediately, especially since it's not an Ember-related thing.

Reply:
Ya, originally i replaced `run.next` with just `run`, but on the demo page popover's with the click event stopped working (only in the demo page, worked fine when using in our apps).

So i switched it back but still needed to run `event.stopPropogation` right away or else it wouldn't actually stop propogating

Reply:
Took the `get('stopPropagation')` outside the handler so we don't need the run loop